CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# v2.9.0 (2024-10-09)
+
+## OS Changes
+* Add nvidia-persistenced and nvidia-modprobe to kmod-*-nvidia ([#122])
```

Comment:
This is not the purpose of the change, this could be just `Run nvidia-persistenced as a system service`